These manpages are installed by wsjtx, remove them from jtdx

--- a/manpages/CMakeLists.txt
+++ b/manpages/CMakeLists.txt
@@ -1,12 +1,12 @@
 set (ASCIIDOC_MANS
   man1/jtdx.1.txt
-  man1/wsprd.1.txt
-  man1/jt65code.1.txt
+  #man1/wsprd.1.txt
+  #man1/jt65code.1.txt
   man1/rigctl-jtdx.1.txt
   man1/rigctld-jtdx.1.txt
   man1/rigctlcom-jtdx.1.txt
-  man1/message_aggregator.1.txt
-  man1/udp_daemon.1.txt
+  #man1/message_aggregator.1.txt
+  #man1/udp_daemon.1.txt
   )
 
 find_program (A2X_EXECUTABLE NAMES a2x a2x.py)
